### DISCUSSION TITLE
Venn diagram

### DISCUSSION THREAD

**Discussion by akshit-sharma** [alandefreitas/matplotplusplus#369] [Ideas] [answered]
How can one plot venn diagram using this library? (if this library supports).

I am looking for venn diagram with 3 different sets.

**Comment by alandefreitas** [comment, marked as answer]
There's no builtin plot type for venn diagrams. You might be able to achieve the same result with circles and text.

https://alandefreitas.github.io/matplotplusplus/annotations/ellipse/
https://alandefreitas.github.io/matplotplusplus/annotations/text/
